src/main/java/org/zalando/nakadi/service/AuthorizationValidator.java

```diff
@@ -185,4 +199,43 @@ public void validateAuthorization(final EventType original, final EventTypeBase
         validateAuthorization(newAuth);
     }
 
+    public AdminAuthorization getAdmins() {
```

Comment:
Why do you think this methods belong to validator? It looks more like service level logic of settigns controller

Reply:
4e4d08a